docs/api/app.md

```diff
@@ -852,6 +852,19 @@ correctly.
 
 **Note:** This will not affect `process.argv`.
 
+### `app.getColorizationColor()` _Windows_
+
+Returns the users current system wide color preference in the form of an ARGB
```

Comment:
Inside the docs?

Perhaps linking to something like this http://www.windowscentral.com/how-custom-accent-color-windows-10 explains what the color is.

On Windows 7 and 8 it is your Aero theme color.  On Windows 10 it's your "theme color".